### PR TITLE
maxcso: init at 1.12.0

### PR DESCRIPTION
**Pull request by david-sawatzke**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/archivers/maxcso/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libuv, lz4, zlib }:
+
+with stdenv.lib;
```
Comment (prusnak) [inline pkgs/tools/archivers/maxcso/default.nix:3]: Let's remove this line
```suggestion
```

File: pkgs/tools/archivers/maxcso/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    repo = "maxcso";
+    rev = "v${version}";
+    sha256 = "10r0vb3ndpq1pw5224d48nim5xz8jj94zhlfy29br6h6jblq8zap";
+  };
+
+  buildInputs = [ libuv lz4 zlib ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = {
```
Comment (prusnak) [inline pkgs/tools/archivers/maxcso/default.nix:19]: And add this here:
```suggestion
  meta = with stdenv.lib; {
```

File: pkgs/tools/archivers/maxcso/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    homepage = "https://github.com/unknownbrackets/maxcso/";
```
Comment (prusnak) [inline pkgs/tools/archivers/maxcso/default.nix:20]: Let's drop the tailing slash:
```suggestion
    homepage = "https://github.com/unknownbrackets/maxcso";
```

File: pkgs/tools/archivers/maxcso/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    description =
+      "A fast ISO to CSO compression program for use with PSP and PS2 emulators, which uses multiple algorithms for best compression ratio";
+    maintainers = [ maintainers.david-sawatzke ];
```
Comment (prusnak) [inline pkgs/tools/archivers/maxcso/default.nix:23]:
```suggestion
    maintainers = with maintainers; [ david-sawatzke ];
```

**Review by prusnak — CHANGES_REQUESTED** [review]

File: pkgs/tools/archivers/maxcso/default.nix (hunk 1 above)
Comment (david-sawatzke) [reply pkgs/tools/archivers/maxcso/default.nix]: Thanks, done

**Review by prusnak — APPROVED** [review]

**Pull request merged**